packages/ui/src/components/evm/erc20-token-picker/index.tsx

```diff
@@ -32,12 +25,11 @@ export interface ERC20TokenPickerProps {
     onDismiss?: () => void;
     loading?: boolean;
     selectedToken?: TokenInfoWithBalance | null;
+    searchQueryObserver?: (query: string) => void;
```

Comment:
Isn't it more consistent to rename this to `onSearchQueryChange`?

Reply:
Fixed in ec5dd5c9a16aab89aac8b5497ad488db8a3ab2ad.